charts/opentelemetry-collector/values.yaml

```diff
@@ -31,6 +31,17 @@ presets:
   # Best used with mode = daemonset.
   hostMetrics:
     enabled: false
+    # adding filesystem field allows you to include and exclude filesystems
+    # filesystem:
+    #   include_devices:
+    #     devices: [ <device name>, ... ]
+    #     match_type: <strict|regexp>
+    #   include_fs_types:
+    #     fs_types: [ <filesystem type>, ... ]
+    #     match_type: <strict|regexp>
+    #   include_mount_points:
+    #     mount_points: [ <mount point>, ... ]
+    #     match_type: <strict|regexp>
```

Comment:
My initial reaction is that this capability is counter to the goal of a preset which is to allow users to enable features without having to have deep k8s/component understanding.  Requiring the user to specify the include/exclude points and know the path names feels like too much knowledge requirement for a preset.

I'd prefer a solution that is more like a switch: "if you enable this then you get X".  Since it is a preset we can make assumptions about what people need and limit the switch to only those needs.  If people require more they can manually configure the component. @povilasv @dmitryax what do you think?

Reply:
If we don't have a good set of assumptions to use to pre-define a list of includes/excludes it may mean that this isn't actually suitable for a preset and users should configure the component themselves. @dmitryax.